### ISSUE TITLE
Automatically watch files

### ISSUE THREAD

**Issue by JaneJeon** [JaneJeon/express-objection-starter#43]
A lot of things need to happen automatically:
- typescript generation from json schema
- json generation from yaml files
- prettier/eslint/whatever

The goal is to make them run automatically, but not unnecessarily:
- linting should happen only before a commit, and shouldn't trigger a restart (i.e. nodemon)
- json generation should happen _every time_ yaml file changes during development, and should be done before running any tests
- typescript generation would be a "nice to have" feature during development, but could be considered part of linting as well.

Ideally, these should be handled by hooks.

Look at https://github.com/chrisvfritz/vue-enterprise-boilerplate for "inspiration"

**Comment by JaneJeon** [comment]
Check out `yarn build:watch` in mjml package, which might be what I want
